misc/start_faux

```diff
@@ -94,6 +88,13 @@ elif [ -n "${options[nobrute]}" ]; then
     (python pentests/brute_server.py good 10000; echo Brute done.) &
 fi
 
+if [ -n "${options[tls]}" ]; then
```

Comment:
is there a negative test case?  There's only "no server" (will fail with connection), or "works as intended" (passes cert check) -- where's the "connected, but not actuallky doing TLS" test case?

Reply:
added an expired certificates version